connect/config.py

```diff
@@ -82,7 +82,7 @@ class Settings(BaseSettings):
 
     # ipfs-cluster
     ipfs_cluster_uri: str = "http://0.0.0.0:9099"
-    ipfs_cluster_replication_factor: int = 2
+    ipfs_cluster_replication_factor: int = -1
```

Comment:
should we add a comment or a "constant" to indicate that we are disabling replication (assuming that is what -1 does)?

Reply:
-1 means replicate everywhere, which covers our single-node cluster.  I can add a comment to that effect.